core/src/main/java/org/apache/iceberg/ManifestFiles.java

```diff
@@ -155,11 +171,30 @@ public static ManifestWriter<DataFile> write(PartitionSpec spec, OutputFile outp
    */
   public static ManifestWriter<DataFile> write(
```

Comment:
Question to the reviewers: should we deprecate this method?

Reply:
No.